app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -57,7 +56,7 @@ def validate_options!(*options)
     end
 
     def show_readme?
-      options.none?(OPTION_NO_README)
+      (options & VALID_OPTIONS).none?
```

Comment:
One thing here - since you folded the OPTION_NO_README into the VALID_OPTIONS list (since it was the only member), this might present a sharp edge in the future, if we ever add a new option - we'd need to split this apart again (the valid options is needed for validation).

If the options list is closed to new addition, maybe naming it "no readme options" makes sense.

Also, I had to stop and think about array intersection to figure out if this was correct (it is, but it made me stop and think). Would saying "none of the options are in the list of known options" make sense?

```ruby
def show_readme?
  options.none? { |option| option.in?(VALID_OPTIONS) } # or VALID_OPTIONS.include?(option)
end
```

The premature micro-optimizer in me ignores the fact that we're comparing a (normally empty, otherwise almost always a singleton) options list with an array of just two acceptable items, and looks for a way to short-circuit that. That part of me could be very wrong.

The evil optimizer in me, leveraging the fact that only the no-readme options are valid, and validate_options! already checked for inclusion, wants to re-write this to just say "options list was empty"

```ruby
def show_readme?
  options.none?
end
```

That's the worst of both worlds (definitely don't listen to that advice!)

Reply:
@djuber I really like your point that the options are already validated before we get to `show_readme?`. Made some changes.